crates/swc_ecma_minifier/tests/terser/compress/hoist_props/issue_2377_2/output.js

```diff
@@ -1,2 +1 @@
-var x;
```

Comment:
This is only safe if `toplevel` is off. Otherwise, this declare a global variable that other scripts can then read without throwing.